src/frontend/komponenter/Fagsak/Mottaker/Verge.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { useState } from 'react';
+
+import styled from 'styled-components';
+
+import { Systemtittel } from 'nav-frontend-typografi';
+
+import { FamilieInput, FamilieKnapp } from '@navikt/familie-form-elements';
+
+import { useMottakerType } from '../../../context/MottakerTypeContext';
+
+const StyledDiv = styled.div`
+    margin: 1rem 0;
+`;
+
+const StyledKnapp = styled(FamilieKnapp)`
+    margin-top: 0.3rem;
+    height: 1rem;
+`;
+
+const StyledFamilieInpunt = styled(FamilieInput)`
+    margin-top: 1.8rem;
+`;
+
+const Verge: React.FunctionComponent = () => {
+    const { hentPerson, lesevisning, registrertVerge, skjema } = useMottakerType();
+    const [spinner, settSpinner] = useState(false);
+
+    if (registrertVerge) {
+        const adresse = registrertVerge.adresse.split('\n').at(0);
+        const postnummerOgSted = registrertVerge.adresse.split('\n').at(1);
+        skjema.felter.fødselsnummer.validerOgSettFelt(registrertVerge.ident || ' ');
```

Comment:
This parsing algorithm seems not very robust. Do we have very strict rule on the input text?

Reply:
It is a decomposition of the submitted text format, but it is probably all just temporary, since we actually don't need the adresse when sending letter using the dokdist-API.